accepted/2020/platform-exclusion/platform-exclusion.md

```diff
@@ -132,14 +270,32 @@ public void Api_Usage()
 ```
 
 ```C#
-// On Windows, the API was unsupported up to and including version 10.0.19041.
+// On Windows, the API was unsupported until version 10.0.19041.
 // The API is considered supported everywhere else without constraints.
-[UnsupportedOSPlatform("windows0.0-10.0.19041")]
+[UnsupportedOSPlatform("windows")]
+[SupportedOSPlatform("windows10.0.19041")]
 public extern void Api();
 
 public void Api_Usage()
 {
-    if (!RuntimeInformation.IsOSPlatformOrEarlier("windows10.0.19041"))
+    if (!RuntimeInformation.IsOSPlatform("windows") ||
+         RuntimeInformation.IsOSPlatformOrLater("windows10.0.19041"))
+    {
+        Api();
+    }
+}
```

Comment:
I think we ran into trouble with `||` checks, right @buyaa-n?  Would this work or would we this need to be expressed differently?

Reply:
Right, it is not supported directly, but with the given API proposal with a combination of attributes its good to have/support this i guess, we can try to support this.